pkg/apis/pipeline/v1beta1/pipelinerun_types.go

```diff
@@ -600,6 +601,11 @@ type PipelineTaskRunSpec struct {
 	StepOverrides []TaskRunStepOverride `json:"stepOverrides,omitempty"`
 	// +listType=atomic
 	SidecarOverrides []TaskRunSidecarOverride `json:"sidecarOverrides,omitempty"`
+	// Compute Resources required by the Task/TaskRun (Pod).
```

Comment:
I'd remove the comment about pods here

Reply:
copy here. will keep as 
`// Compute Resources required by the Task/TaskRun`

Reply:
closed - removed words on `Pod`